Lib/pprint.py

```diff
@@ -179,6 +185,9 @@ def _format(self, object, stream, indent, allowance, context, level):
         max_width = self._width - indent - allowance
         if len(rep) > max_width:
             p = self._dispatch.get(type(object).__repr__, None)
+            # Lazy import to improve module import time
+            import dataclasses as _dataclasses
```

Comment:
I can't reach L196 in the suggestion, but that's the other half
```suggestion
            # Lazy import to improve module import time
            from dataclasses import is_dataclass
```